rds/sendrequest.go

```diff
@@ -0,0 +1,27 @@
+package rds
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+)
+
+func SendRequest(sql string, url string) (string, int, error) {
```

Comment:
we probably gonna need some:

req.Header.Set("Authorization", key) here right?

Reply:
Dapat... Will put in once endpoint's testable.